apierrors/errors.go

```diff
@@ -31,7 +33,13 @@ var (
 		ErrInsertedObservationsInvalidSyntax: true,
 		ErrMissingJobProperties:              true,
 		ErrMissingParameters:                 true,
+		ErrInvalidPositiveInteger:            true,
 		ErrUnableToParseJSON:                 true,
 		ErrUnableToReadMessage:               true,
 	}
 )
+
+// ErrorMaximumLimitReached creates an for the given limit
+func ErrorMaximumLimitReached(m int) error {
+	return fmt.Errorf("the maximum limit has been reached, the limit cannot be more than %s", strconv.Itoa(m))
```

Comment:
No need to cast int to string you can use the fmt placeholder `%d` for integer types:
```
return fmt.Errorf("the maximum limit has been reached, the limit cannot be more than %d",  m)
```